### DISCUSSION TITLE
Whitelisting

### DISCUSSION THREAD

**Discussion by Avijeet-Blocsol** [ONDC-Official/developer-docs#47] [Q&A]
Hello @ONDC Team, I have applied for the whitelisting of my domain for ONDC financial services but have not yet received any response to myemail from ONDC. My email is avijeet@invoicepe.in and the domain was ondc.invoicepe.in. Please help me resolve this issue.

**Comment by Avijeet-Blocsol** [comment]
It has been 7 days, I have been waiting for a reply from your side. Please help me resolve this issue ASAP.

**Comment by Avijeet-Blocsol** [comment]
Any updates on my last message? Please let me know any specific email Id where I should request ONDC if this is not the right platform. I need to resolve this issue.

**Comment by pratikmazumdar** [comment]
Hey, Avi looking into your issue.

**Comment by pratikmazumdar** [comment]
It seems that we didn't received your subscriber id, can you apply for whitelisting again. Do let us know if you didn't get whitelisted within 3-4 hours

**Comment by pratikmazumdar** [comment]
I will be closing this issue as there was no response from your side.
